### DISCUSSION TITLE
Welcome to head-pose-estimation Discussions!

### DISCUSSION THREAD

**Discussion by yinguobing** [yinguobing/head-pose-estimation#37] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by yinguobing** [comment]
I think this is a good replacement for issues when most people are too afraid to open one. And the topics here are obviously more open.

**Reply by Wxy778899** [reply]
您好，请问yaw最大检测角度能达到多少？

**Reply by yinguobing** [reply]
我没有测过，估计不到30度。

**Reply by Wxy778899** [reply]
thank 
you
